homework.backgrounds(opt)/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+}
+
+
+.forest {
+    width: 100%;
+    height: 100%;
+    background-size: cover;
+    position: relative;
+}
+
+.overlay {
```

Comment:
As sugera ca width-ul si height-ul sa fie 100% in cazul la overlay si sa pui valorile fixe pe parinte (forest)